purchase_work_acceptance/models/purchase.py

```diff
@@ -19,6 +19,9 @@ class PurchaseOrder(models.Model):
         string="WA Lines",
         readonly=True,
     )
+    invisible_create_wa = fields.Boolean(
```

Comment:
```suggestion
    wa_accepted = fields.Boolean(
```
And add help = All quantity in purchase order are fully accepted.

Reply:
This field can be more useful with this,
![image](https://user-images.githubusercontent.com/1973598/105498382-b432c800-5cf2-11eb-8206-7ab6e6d6a9cf.png)

Reply:
fixed